components/script/script_module.rs

```diff
@@ -1118,6 +1119,35 @@ unsafe extern "C" fn HostResolveImportedModule(
     unreachable!()
 }
 
+#[allow(unsafe_code, non_snake_case)]
+/// https://tc39.es/ecma262/#sec-hostgetimportmetaproperties
+/// https://html.spec.whatwg.org/multipage/#hostgetimportmetaproperties
+unsafe extern "C" fn HostPopulateImportMeta(
+    cx: *mut JSContext,
+    reference_private: RawHandleValue,
+    meta_object: RawHandle<*mut JSObject>,
+) -> bool {
+    let in_realm_proof = AlreadyInRealm::assert_for_cx(SafeJSContext::from_ptr(cx));
+    let global_scope = GlobalScope::from_context(cx, InRealm::Already(&in_realm_proof));
+
+    // Step 2.
+    let base_url = match (reference_private.to_private() as *const ModuleScript).as_ref() {
+        Some(module_data) => module_data.base_url.clone(),
+        None => global_scope.api_base_url(),
+    };
+
+    rooted!(in(cx) let url_string = JS_NewStringCopyZ(cx, base_url.as_str().as_ptr() as *const i8));
+
+    // Step 3.
+    JS_DefineProperty4(
+        cx,
+        meta_object,
+        "url".as_ptr() as *const i8,
+        url_string.handle().into_handle(),
+        JSPROP_ENUMERATE.into(),
+    )
```

Comment:
@jdm I've followed Gecko to use those functions from mozjs but I still can't pass the import meta test which means the `import.meta` doesn't have the `url` attribute.

Do I miss something to set the property 🤔?

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6782666/82111754-3ccf9080-9782-11ea-89f0-c3a75a5b348b.png">